File: code-kata/Absolute-beginner/Area_of_equilateral_triangle.py
#  The area of an equilateral triangle is ¼(√3a2) where "a" represents a side of the triangle. You are provided with the side "a". Find the area of the equilateral triangle.


import math
a=float(input())
s=math.sqrt(3)
print(round((1/4)*a*a*s,2),end="")

File: javascript/codekata/area_of_equilateral_triangle.js
// Find the area of the equilateral triangle and print the answer up to 2 decimal places after rounding off.

const readline=require('readline');
const inp=readline.createInterface({
  input:process.stdin});
const n=[];
inp.on("line",(data)=>{
  n.push(data);});
inp.on("close",()=>{
  var a=n[0],s,t;
  s=Math.sqrt(3)
  t=(1/4)*a*a*s
  console.log(t.toFixed(2))
});
